test/README.md

```diff
@@ -0,0 +1,60 @@
+# Guidelines for Writing Tests
+
+## Assumptions & Requirements
+
+1. Directory paths should be well formed such that they can be run from any directory.
+1. Test scripts in the `$HOME/test` directory assume they are run against a functional Openshift cluster that was setup prior to the execution of the test script.
+1. Each script in `$HOME/test` should has a companion script in `$HOME/hack/testing`
+1. Each script in `$HOME/test` performs cleanup logic to reset the state of the cluster and logging stack
+
+```
+#!/bin/bash
+
+source "$(dirname "${BASH_SOURCE[0]}" )/../hack/lib/init.sh"
+
+os::util::environment::use_sudo
+
+os::test::junit::declare_suite_start "test/<TESTNAME>"
+
+if [ -n "${DEBUG:-}" ] ; then
```

Comment:
there is a `$OS_DEBUG` var that turns on printing from `os::log::debug` functions, use instead